### DISCUSSION TITLE
Feature Request: Ability to filter by row group index when reading from Parquet file

### DISCUSSION THREAD

**Discussion by aley1** [duckdb/duckdb#5814] [Ideas]
I will like to suggest a new feature that allow users to specific which row groups to read from (or skip) before applying Select/Where queries on the data.

Example:
SELECT columnA FROM read_parquet('myfile.parquet', row_group=1,5,8,10,15) WHERE columnB='something';

This will instruct the parquet reader to only fetch the specific row groups before performing the query. This opens up the possibility of maintaining external secondary indexes over row group data stored in parquet files.

**Comment by hannes** [comment]
Happy to review a PR
